labelbox/schema/data_row_metadata.py

```diff
@@ -125,6 +138,9 @@ def _build_ontology(self):
         self.reserved_by_name: Dict[
             str,
             DataRowMetadataSchema] = self._make_name_index(self.reserved_fields)
```

Comment:
This old type hint is kind of wrong. Maybe update it to show how it contrasts with your new easier to use function.